src/ipc/server.rs

```diff
@@ -92,6 +116,22 @@ pub enum RpcHandle {
         id: Id,
         x: Destory,
     },
+    StatIndexing {
```

Comment:
I think we need to monitor status of all segments. It's Okay to leave it for future.